### DISCUSSION TITLE
What are sender-key & session-** files?

### DISCUSSION THREAD

**Discussion by bobftool** [WhiskeySockets/Baileys#304] [General Questions]
I have this as my sock:

`const { state, saveCreds } = await useMultiFileAuthState('baileys_auth_info')`

`const sock = makeWASocket({
		defaultQueryTimeoutMs: undefined,
		printQRInTerminal: true,
		auth: {
			creds: state.creds,
			/** caching makes the store faster to send/recv messages */
			keys: makeCacheableSignalKeyStore(state.keys, logger),
		},
		generateHighQualityLinkPreview: true
	})`

And everytime someone sends a message to the bot or a group it creates a file in _baileys_auth_info_ folder with the phone number of the sender included in the name of that file.

![Captura de pantalla 2023-08-01 030616](https://github.com/WhiskeySockets/Baileys/assets/112662580/9d48a261-030e-482a-8594-0da509140e7b)

I want to know what is causing this and how to disable it, because I don't want to fill up disk space with all these files.

**Comment by Riders004** [comment]
It's a session file for user those who are chatted with you
